crates/sozo/src/commands/options/account.rs

```diff
@@ -33,6 +33,7 @@ pub struct AccountOptions {
     #[arg(value_name = "PATH")]
     #[arg(help_heading = "Signer options - KEYSTORE")]
     #[arg(help = "Use the keystore in the given folder or file.")]
+    // TODO: Should we add `DOJO_KEYSTORE_PATH_ENV_VAR`?
```

Comment:
Makes sense to have a unified experience for someone that wants to have everything set from environment variables.
`DOJO_KEYSTORE` may be enough?

Reply:
to be consistent with: https://github.com/dojoengine/dojo/blob/79278a62f66895cbe3bfaf822e3703309f8847b5/crates/sozo/src/commands/options/mod.rs#L8

i was thinking `DOJO_KEYSTORE_PATH`

Reply:
> to be consistent with: https://github.com/dojoengine/dojo/blob/79278a62f66895cbe3bfaf822e3703309f8847b5/crates/sozo/src/commands/options/mod.rs#L8
> 
> 
> 
> i was thinking `DOJO_KEYSTORE_PATH`

Oh I see, if it makes sense to you let's keep it like this yes. 👍 
In my head the keystore IS a file that's why I proposed that, but maybe more explicit your way.

> that `_ENV_VAR` end is just in sozo code to be more descriptive.

Yep all good for this one.